osu/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full-vnc
+
+# Install Prerequisites
+COPY compile /tmp
```

Comment:
you would need to, provide the `context` property in the `.gitpod.yml` to have access to the compile here.